### PR TITLE
Rego policy draft blog post

### PR DESCRIPTION
**Pull request by vjanelle**

### CODE REVIEW

**Comment by vjanelle** [comment]
@ripienaar I want to enhance this somehow, possibly by adding the time of day functions to a policy example.

**Comment by vjanelle** [comment]
* make callerid and provisioning_mode it’s more consistent with other makings - callerid is in aaasvc like that 
* mention the puppet module additions

**Comment by vjanelle** [comment]
Waiting for https://forge.puppet.com/choria/mcollective to get released

**Comment by ripienaar** [comment]
https://forge.puppet.com/choria/mcollective/0.10.3

File: blog/content/post/2020/02/14/rego_policies_opa.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: "Rego policies for Choria Server"
+date: 2020-02-14T09:00:00+07:00
+tags: ["security", "opa", "rego", "actionpolicy"]
+draft: true
```
Comment (ripienaar) [inline blog/content/post/2020/02/14/rego_policies_opa.md:5]: Do a update here, set draft false and add author like here:

https://github.com/choria-io/website/blame/b44bac23169b8aced9c724d586a5285dce625143/blog/content/post/2019/10/01/python_intro.md#L6

**Pull request merged**